api.py

```diff
@@ -0,0 +1,76 @@
+import os
+import logging
+import requests
+import flask
+import datetime
+
+
+from flask import Flask, request, flash, redirect, render_template, url_for, jsonify
+from flask_sqlalchemy import SQLAlchemy
+
+app = Flask(__name__)
+app.config.update(dict(
+    SQLALCHEMY_DATABASE_URI='sqlite:///kraken.db',
+    DEBUG=True,
+    SECRET_KEY='development',
+    LOG_LEVEL=logging.DEBUG,
+    ENVIRONMENT='development'
+))
+app.config.from_envvar('FLASK_SETTINGS', silent=True)
+
+logging.basicConfig(level=logging.DEBUG, filename='logs/' + app.config['ENVIRONMENT'] + '.log')
+
+db = SQLAlchemy(app)
+
+# TODO: reused model from sesh-dash! DRY!!! - at least move database stuff it into a separate file
+class Sesh_Site(db.Model):
+    __tablename__= 'sites'
+    id = db.Column(db.Integer, primary_key=True)
+    site_name = db.Column(db.String(100))
+
+    def __str__(self):
+        return self.site_name
+
+class BoM_Data_Point(db.Model):
```

Comment:
I'm hesitant to have this defined twice. Meaning that if the model in sesh-dash changes this needs to be changed as well.  Is it worth using submodule here to import from sesh-dash?